### PR TITLE
Eg xxx task list refactor

### PR DESCRIPTION
**Pull request by LesWilson**
This is tied in with associated changes to external-guidance and alsoo journey-tests

### CODE REVIEW

File: app/views/fact_check_content_review.scala.html [hunk 1]
```diff
@@ -15,13 +15,21 @@
...
 @this(layout: main_layout, govukTable : GovukTable, govukBackLink : GovukBackLink)
 
 @(review: ApprovalProcessReview)(implicit request: Request[_], messages: Messages)
 
+@reviewStatus() = @{
+  val totalPages = review.pages.length
+  val incompletePages = review.pages.count(p => p.status == PageReviewStatus.NotStarted)
+  incompletePages match {
+    case 0 => "Review Complete - all pages checked"
+    case _ => s"Review Incomplete - ${totalPages - incompletePages} of $totalPages pages checked"
```
Comment (cmrhmrc) [inline app/views/fact_check_content_review.scala.html:29]: Wouldnt it be easier to count the complete pages, as thats what you are going to display, saves having to calculate the number later

File: app/controllers/FactCheckPageReviewController.scala [hunk 2]
```diff
@@ -69,7 +69,7 @@ class FactCheckPageReviewController @Inject() (
       .fold(
         (formWithErrors: Form[FactCheckPageReview]) => { Future.successful(BadRequest(view(processId, s"/$page", formWithErrors))) },
         result => {
-          val reviewDetail = PageReviewDetail(processId, s"/$page", Some(result.answer), Complete, updateUser = Some(s"${request.credId}:${request.name}"))
+          val reviewDetail = PageReviewDetail(processId, s"/$page", "", Some(result.answer), Complete, updateUser = Some(s"${request.credId}:${request.name}"))
```
Comment (cmrhmrc) [inline app/controllers/FactCheckPageReviewController.scala:72]: The re-use here with blanks args suggests we need two case classes for the different situations

File: app/views/twoeye_content_review.scala.html [hunk 3]
```diff
@@ -16,13 +16,21 @@
...
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 
 @this(layout: main_layout, govukTable : GovukTable, govukBackLink : GovukBackLink, appConfig: AppConfig)
 
 @(review: ApprovalProcessReview)(implicit request: Request[_], messages: Messages)
 
+@reviewStatus() = @{
+  val totalPages = review.pages.length
+  val incompletePages = review.pages.count(p => p.status == PageReviewStatus.NotStarted)
+  incompletePages match {
```
Comment (cmrhmrc) [inline app/views/twoeye_content_review.scala.html:28]: as before

File: test/views/TwoEyeReviewSpec.scala [hunk 4]
```diff
@@ -60,19 +60,19 @@ class TwoEyeReviewSpec extends ViewSpecBase {
...
-        h2s(1).text shouldBe s"2. ${messages("2iReview.pagesReviewedConfirmationHeading")}"
-
-        val uls = ol.getElementsByTag("ul").asScala.toList
-        uls.size shouldBe 2
-
-        Option(uls(1).getElementsByTag("a").first).fold(fail("Missing Send confirmation link")) { a =>
-          a.text shouldBe messages("2iReview.sendConfirmation")
-        }
-      }
+//      Option(doc.getElementsByTag("ul").first).fold(fail("Missing unordered list elem (ul)")) { ol =>
```
Comment (cmrhmrc) [inline test/views/TwoEyeReviewSpec.scala:63]: Can delete this

File: app/views/fact_check_content_review.scala.html (hunk 1 above)
Comment (LesWilson) [reply app/views/fact_check_content_review.scala.html:29]: good point - I was burning the midnight oil when refactoring :)

File: app/controllers/FactCheckPageReviewController.scala (hunk 2 above)
Comment (LesWilson) [reply app/controllers/FactCheckPageReviewController.scala:72]: yes, I said that in the slack chat as I really didn't have time to introduce a whole new class and set of tests.  I think this is something for post accessibility audit

File: app/views/twoeye_content_review.scala.html (hunk 3 above)
Comment (LesWilson) [reply app/views/twoeye_content_review.scala.html:28]: done - in next commit

File: test/views/TwoEyeReviewSpec.scala (hunk 4 above)
Comment (LesWilson) [reply test/views/TwoEyeReviewSpec.scala:77]: done - in next commit

**Pull request closed without merging**